libs/csm/rest/csm_rest_system_health.py

```diff
@@ -626,3 +626,54 @@ def cluster_operation_signal(
         self.log.info("%s operation on %s POST REST API response : %s",
                       operation, resource, response)
         return True, response
+
+    @RestTestLib.authenticate_and_login
+    @RestTestLib.rest_logout
+    def post_resource_signal(
+            self,
+            req_body: dict,
+            resource: str):
+        """
+        This method POST resource failure/shutdown event/signal to cluster
+        :param req_body: POST operation request body
+        :param resource: POST request body info
+        :return: bool, POST API response
+        """
+        # Building request url to POST resource failure signal
```

Comment:
Here can we have logger for what status is being set for which resource?

Reply:
> Here can we have logger for what status is being set for which resource?

updated  the same in HA lib.